### PR TITLE
Add corrections to distributed simulation doc

### PR DESCRIPTION
**Pull request by Antho1426RR**
I brought small typing adjustments here and there in the "distributed simulation" page of RapyutaSimulationPlugins’s documentation.

### CODE REVIEW

File: docs/source/distributed_simulation.rst [hunk 1]
```diff
@@ -87,27 +89,27 @@ Basic flows between client and server
...
 | **(b-4)** SimulationState spawn the robot in the server.
-| **(b-5)** Spawned robot is replicated to the clients. Only In the client which send spawn request, robot create/initiate ROS2 compoenent. No ROS components are created/initiated in other clients.
+| **(b-5)** Spawned robot is replicated to the clients. Only in the client which sends spawn request, robot creates/initiates ROS2 compoenent. No ROS components are created/initiated in other clients.
 
 
 | **Robot Movement flow**
 | **(c-1)** ROS2 application publish command topic such as /cmd_vel
 | **(c-2)** Robot in the client moves to follow command.
-| **(c-3)** Set server robot movement is requested to  `ARRNetworkPlayerController <doxygen_generated/html/db/d54/class_a_r_r_network_player_controller.html>`_ 
+| **(c-3)** Set server robot movement is requested by `ARRNetworkPlayerController <doxygen_generated/html/db/d54/class_a_r_r_network_player_controller.html>`_
```
Comment (yuokamoto) [inline docs/source/distributed_simulation.rst]: Robot requests to set server robot movment to ARRNetworkPlayerController

maybe better? I don't have confidence in English but point is Robot send requests to NetworkPlayercontroller

File: docs/source/distributed_simulation.rst [hunk 2]
```diff
@@ -50,19 +51,19 @@ RPCs and Replication
 
 Communication between Server and Client uses UE’s `replication and RPCs <https://docs.unrealengine.com/4.27/en-US/InteractiveExperiences/Networking/Actors/>`_.
 
-**Replication** is the process to sync data between the server and the clients. Changes of Actors and properties which set as "replciated" in server are synced to the cliets.
-Replciation is the process from server to process to sync data such as Actor pose, and other properties.
+**Replication** is the process to sync data between the server and the clients. Changes of Actors and properties set as "replicated" in server are synced to the clients.
+Replication is the process of the server to sync data such as Actor pose, and other properties.
```
Comment (yuokamoto) [inline docs/source/distributed_simulation.rst]: I want to emphasize Replication is a one-way process from server to client

File: docs/source/distributed_simulation.rst [hunk 3]
```diff
@@ -24,20 +24,21 @@ Architecture Overview
...
-In the robotics simulator, we put robots in the each client. 
+UE multiplayer is based around the client-server model. Each player is a client connected to a single server.
+In the robotics simulator, we put robots in each client. 
 
-.. image:: images/overview_of_UE_distributed_simulation.png
+.. figure:: images/overview_of_UE_distributed_simulation.png
 
-Fig shows an overview of the configuration. 
-The simulator consists of four types of instances. 
+   Figure 1: Overview of the configuration.
```
Comment (yuokamoto) [inline docs/source/distributed_simulation.rst:33]: is it possible to centering the caption?

**Review by yuokamoto — COMMENTED** [review]
I updated readthedocs to build doc from your branch.
https://rapyutasimulationplugins.readthedocs.io/en/docs_corr_distributed_sim/distributed_simulation.html

File: docs/source/distributed_simulation.rst (hunk 1 above)
Comment (Antho1426RR) [reply docs/source/distributed_simulation.rst]: Okay, this is very clear from Figure 2. You're right then I would propose "(c-3) Robot requests ARRNetworkPlayerController to set server robot movement".

File: docs/source/distributed_simulation.rst (hunk 2 above)
Comment (Antho1426RR) [reply docs/source/distributed_simulation.rst]: Yes, then I would propose for the last sentence: "Replication is a one-way process from server to client to sync data such as Actor pose and other properties."

File: docs/source/distributed_simulation.rst (hunk 3 above)
Comment (Antho1426RR) [reply docs/source/distributed_simulation.rst:33]: Yes sure, with the option `:align: center`. Unfortunately, this is not feasible with the TurtleBot video...

**Review by yuokamoto — APPROVED** [review]

**Review by larryng-rapyuta — APPROVED** [review]

**Pull request merged**